### DISCUSSION TITLE
Formatieren Abfrageergebnis

### DISCUSSION THREAD

**Discussion by saltdorfer** [e-netze/webgis-community#17] [Q&A]
Bei der Ausgabe von Spalten kann ja mittels Expression ein Rundung der Zahl oder sogar Umrechnungen usw... definieren, z.B. $round2([Fläche]).
Gibt es hier auch einen Befehl damit die Zahl mit Trennzeichen (10.000) ausgegeben wird? 
Finde da irgendwie nichts, das funktioniert.

**Comment by gis-eni** [comment]
Ich wußte gar nicht, dass diese Methoden noch genutzt werden ;)
Tausenderpunkte gibt es monentan noch nicht,

Aber wenn wir diese Expressions noch genutzt werden, sollte man das einmal komplett neu aufsetzten und sie Wünsche sammeln, die es da noch so geben könnte.

Schreibt hier einmal einmal eure Wünscht ans Christkind, die ihr euch bei den Tabellen Feld Expressions so wünschen würdet!

**Comment by saltdorfer** [comment]
**Wunschliste:**

- 1.000 Trennzeichen
- $round
- $eval([FLAECHE]/10000)
- [PixelValue] => Expressions auch bei AGS ImageServer

hab ich  mal auf die schnelle gefunden ;)

**Reply by gis-eni** [reply]
[PixelValue] => Expressions auch bei AGS ImageServer 
Muss noch umgesetzt werden

**Reply by saltdorfer** [reply]
> [PixelValue] => Expressions auch bei AGS ImageServer Muss noch umgesetzt werden

Weil wir gerade versuchen die Nachkommastellen bei einem Image-Server-Dienst zu reduzieren ;) wurde das eigentlich umgesetzt? Wenn ja wo im CMS definier ich denn die Expression.
